Graph Explorer docs update (#17622)
microsoftgraph/microsoft-graph-docs: concepts/graph-explorer/graph-explorer-overview.md

:::image type="content" source="./images/graph-explorer-screen.png" alt-text="Screenshot of the Graph Explorer user interface." border="true":::

Use Graph Explorer to:

- Try out Microsoft Graph APIs.
- Learn about the permissions required for the different APIs.
- Explore all the resources available on Microsoft Graph.
- Explore Microsoft Graph Toolkit components, adaptive cards and code snippets for your queries.

Graph Explorer handles the authentication process for you. Customize the experience by collapsing the sidebar or changing the theme.

## Get started

Graph Explorer is a web application hosted on the [Microsoft Graph developer center](https://developer.microsoft.com/graph). It's an open-source project, and we welcome your contributions and feedback on [GitHub](https://github.com/microsoftgraph/microsoft-graph-explorer-v4).

### Make requests

> With Graph Explorer, you can make requests to the Microsoft Graph APIs to retrieve, add, delete and update data. Your requests can send parameters, authorization details, and any body data you require.

Reviewer: "... and any body data you require." This phrase may be hard to understand for non-native speakers. 

Suggestion:
```suggestion
With Graph Explorer, you can make requests to the Microsoft Graph APIs to retrieve, add, delete and update data. Your requests can send parameters, authorization details, and any object you require.
```

Author: @isvargasmsft what's referred to here is the request body so it is data specific to that body not just any object.

> :::image type="content" source="./images/request-headers.png" alt-text="Screenshot of request headers in Graph Explorer." border="true":::

Reviewer: GLOBAL: What is the goal of adding the image to the document? Consider this when composing the alt text. The alt text should meet the same goal as the image. If the objective is to illustrate the operation, ask yourself if the alt text is providing enough clarity to meet that objective.

## Retrieve data in Graph Explorer

> To run a GET request in Graph Explorer, you don't have to be signed in. You can retrieve sample data from the default sample tenant.

Reviewer: In general, we avoid passive voice when possible. Guidance here: https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=25523

```suggestion
To run a GET request in Graph Explorer, you don't have to sign in. You can retrieve sample data from the default sample tenant.
```

:::image type="content" source="./images/get-request.png" alt-text="Screenshot of a sample request in Graph Explorer." border="true":::

> To create the request, you can either select a sample query which fills in the query field or you can choose to manually type your request in the field.

Reviewer: Guide your user as if it were their first time in the tool. Example:
```suggestion
To create the request, you can either select a sample query from the menu at the left, which fills in the query field, or you can choose to manually type your request in the field.
```
